dashboard/db/schema.rb

```diff
@@ -555,6 +557,15 @@
   add_index "puzzle_ratings", ["script_id", "level_id"], name: "index_puzzle_ratings_on_script_id_and_level_id", using: :btree
   add_index "puzzle_ratings", ["user_id", "script_id", "level_id"], name: "index_puzzle_ratings_on_user_id_and_script_id_and_level_id", unique: true, using: :btree
 
+  create_table "school_districts", force: :cascade do |t|
+    t.string   "name",       limit: 255
+    t.string   "city",       limit: 255
+    t.string   "state",      limit: 255
+    t.string   "zip",        limit: 255
+    t.datetime "created_at",             null: false
+    t.datetime "updated_at",             null: false
```

Comment:
Oh, I guess creating a migration automatically created these.  They are ActiveRecord features, and seem to feature in most of our models:

http://guides.rubyonrails.org/active_record_basics.html#schema-conventions